core/src/test/java/io/grpc/internal/ServiceConfigErrorHandlingTest.java

```diff
@@ -153,7 +155,18 @@ public ConfigOrError parseLoadBalancingPolicyConfig(
   private ObjectPool<Executor> balancerRpcExecutorPool;
   @Mock
   private Executor blockingExecutor;
-  private ChannelBuilder channelBuilder;
+
+  private ManagedChannelImplBuilder channelBuilder = new ManagedChannelImplBuilder(TARGET,
+      new ClientTransportFactoryBuilder() {
+        @Override public ClientTransportFactory buildClientTransportFactory() {
+          throw new UnsupportedOperationException();
+        }
+      },
+      new ChannelBuilderDefaultPortProvider() {
```

Comment:
It seems a hard-coded port is fairly common. It might be fair to create a `ManagedChannelImplBuilder.FixedPortProvider` (or as a top-level class in core/src/test/io/grpc/internal) which just accepts the port in its constructor. Then you'd use `new FixedPortProvider(DEFAULT_PORT)`.

Not important, but `Fixed*` classes for stuff like this is semi-common in our code.